pkgs/dart_tooling_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -606,6 +625,100 @@ base mixin DartToolingDaemonSupport on LoggingSupport, ToolsSupport {
       ),
     ],
   );
+
+  static final runtimeErrorsScheme = 'runtime-errors';
+}
+
+/// Listens on a VM service for errors.
+class _AppErrorsListener {
```

Comment:
nit: rename to _FlutterAppErrorsListener to make it clear this is only for flutter apps and not pure dart apps